tests/phpunit/includes/TestCase.php

```diff
@@ -71,6 +71,10 @@ function ( $status, $location ) { // phpcs:ignore WordPressVIPMinimum.Hooks.Alwa
 			10,
 			2
 		);
+
+		// Initialize the global $wp_scripts and $wp_styles, which have been cleared in tear_down.
```

Comment:
Total nitpick, the first time this runs they won't yet have been cleared in `tear_down()`.

```suggestion
		// Initialize the global $wp_scripts and $wp_styles, which are cleared in tear_down.
```